modules/bday.js

```diff
@@ -0,0 +1,227 @@
+        },
+      },
+    ],
+  })
+)
+const confirmChannel = (response_url, cmd, channel) => (
+  axios.post(response_url, {
+    response_type: 'ephemeral',
+    blocks: [
+      { type: 'divider' },
+      {
+        type: 'section',
+        text: {
+          type: 'mrkdwn',
+          text: `Please run the */bday ${cmd}* command in channel: #${channel}.`,
+        },
+      },
+    ],
+  })
+)
+
+const worker = async ({ command, channel_id, channel_name, response_url }) => {
+  // send card for members to sign
+  if (command.includes('sign') && command.includes('for')) {
+    // make sure to execute command from #general
+    if (channel_name !== 'general') {
+      return confirmChannel(response_url, 'sign', 'general')
+    }
+    // sign URL for @user_id|user_name
+    const R = /<(?<MIRO_URL>.*)> for <@(?<BDAY_USER_ID>.*)\|(?<BDAY_USER_NAME>.*)>/
+    const matches = command.match(R)
+    // notify user of invalid input
+    if (!matches) {
+      return invalidInputNotice(response_url, command)
+    }
+    // send to every member in #general excluding bday person
+    const { groups: { MIRO_URL, BDAY_USER_ID, BDAY_USER_NAME } } = matches
+    const { members } = await web.conversations.members({ channel: channel_id })
+    members.splice(members.findIndex((m) => m === BDAY_USER_ID), 1)
+    return Promise.all(
+      members.map(async (channel) => await web.chat.postMessage({
+        channel,
+        blocks: [
+          { type: 'divider' },
+          {
+            type: 'section',
+            text: {
+              type: 'mrkdwn',
+              text: [
+                `:tada: *${BDAY_USER_NAME}'s* birthday is coming up soon! :birthday:`,
+                `:tada: When you get a chance, visit ${MIRO_URL} to sign their birthday card!`,
+              ].join('\n'),
+            },
+          },
+        ],
+      }))
+    )
+      .then(() => {
+        // notify user invitation has been sent
+        return axios.post(response_url, {
+          response_type: 'ephemeral',
+          blocks: [
+            { type: 'divider' },
+            {
+              type: 'section',
+              text: {
+                type: 'mrkdwn',
+                text: `Invitation to sign on ${MIRO_URL} has been sent to members in #${channel_name}!`,
+              },
+            },
+          ],
+        })
+      })
+      .catch((e) => {
+        return axios.post(response_url, {
+          response_type: 'ephemeral',
+          text: `Something went wrong: ${e}.`,
+        })
+      })
+  }
+
+  // send card to bday user
+  if (command.includes('send') && command.includes('to')) {
+    const R = /<(?<MIRO_URL>.*)> to <@(?<BDAY_USER_ID>.*)\|(?<BDAY_USER_NAME>.*)>/
+    const matches = command.match(R)
+    // notify user of invalid input
+    if (!matches) {
+      return invalidInputNotice(response_url, command)
+    }
+    // send to bday user
+    const { groups: { MIRO_URL, BDAY_USER_ID, BDAY_USER_NAME } } = matches
```

Comment:
I feel like a few steps are repeated, maybe could do them before checking if it's `send|sign` since they are the same for both